bin/hdfs-mesos-namenode

```diff
@@ -18,23 +18,17 @@ fi
 trap "{ $DIR/mesos-killtree "$$" ; exit 0; }" EXIT
 
 function bootstrap_standby() {
-  echo "Pausing for 10s before bootstrapping standby namenode"
```

Comment:
The other sleeps seem necessary. Within `hdfs-mesos-namenode/journalnode/zkfc`, the `sleep 10` inside the `while [ true ]` loop guarantees things have settled down after the process exited before restarting it. The sleep in `mesos-killtree` gives the processes being killed time for a graceful shutdown between `SIGTERM` and `SIGKILL`.
So these are the only ones slowing down namenode (cluster) startup unnecessarily.